### PR TITLE
Bug 1172465 - border-radius should be applied to all buttons

### PR DESCRIPTION
**Pull request by ayushagrawal288**
Replaced marketplace-elements.styl file with forms.styl

### CODE REVIEW

File: src/media/css/newsletter.styl [hunk 1]
```diff
@@ -204,9 +204,9 @@ $arrow-size = 10px;
     }
     .newsletter-signup-button,
     .newsletter-cancel-button {
-        width: auto;
-        margin: 0;
         border-radius: 5px;
+        margin: 0;
+        width: auto;
```
Comment (gurumukhi) [inline src/media/css/newsletter.styl]: You could have avoided adding these changes from diffs I think, but it should be fine for now.

**Comment by ngokevin** [comment]
This doesn't work well on Desktop:

screen shot 2015-08-26 at 11 32 28 am

I would limit these changes only to the Accounts page buttons.

File: src/media/css/forms.styl [hunk 2]
```diff
@@ -12,6 +12,7 @@ button, input, select, textarea {
 }
 
 button, input {
+    border-radius: 5px;
```
Comment (tsl143) [inline src/media/css/forms.styl]: Instead try following 

label, button, input[type=button], input[type=submit] {
    border-radius: 5px;
    cursor: pointer;
}
[data-modal] button{
    border-radius: 0;
}

data-modal will take care of all popup border radius,

**Comment by spasovski** [comment]
It seems this is no longer an issue on desktop or mobile. The bug can be closed as "worksforme". Thanks.

**Pull request closed without merging**